### ISSUE TITLE
v_prediction high loss

### ISSUE THREAD

**Issue by rmac85** [huggingface/diffusers#2519]
Hello, i recently tried dreambooth fine-tuning using v_prediction instead of epsilon. I noticed that the loss was much higher with this method. With epsilon the loss is usually steady between 0.25 - 0.3 but with v_prediction it went up to 1.5 - 2, eventually dropping but not any lower than 0.8. Just wondering if this is normal behaviour for this prediction type or perhaps I could be missing a step. Used a learning rate of 9e-7.

scheduler_config.json:
```
{
  "_class_name": "DDPMScheduler",
  "_diffusers_version": "0.13.0.dev0",
  "beta_end": 0.02,
  "beta_schedule": "scaled_linear",
  "beta_start": 0.0001,
  "num_train_timesteps": 1000,
  "prediction_type": "v_prediction",
  "variance_type": "learned"
  "trained_betas": null
}
```

**Comment by patrickvonplaten** [comment]
Hey @rmac85,

Thanks for the issue! We've recently decided to use the forum for such questions (more training-related, less bug-related).
Could you try to pose your question there again: https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63
